client/my-sites/plugins-wpcom/premium-plugins-panel.jsx

```diff
@@ -26,9 +30,10 @@ export const PremiumPluginsPanel = React.createClass( {
 
 				<Card className={ cardClasses }>
 					<div className="wpcom-plugins__list">
-						{ plugins.map( ( { name, descriptionLink, icon, plan, description } ) =>
-							<PremiumPlugin
-								{ ...{ name, key: name, descriptionLink, icon, plan, description } }
+						{ plugins.map( ( { name, descriptionLink, icon, category, description } ) =>
+							<Plugin
+								onClick={ () => onClick( name ) }
```

Comment:
:) The fat arrow warnings are non-blocking, but its just nice if we can avoid them.

Another alternative is to let the plugin provide an interface, where the plugin prop values are provided, and we pass through a trackClick function.

In Plugin:
```
onClick() {
     this.props.trackClick( this.props );
     this.props.onClick();
}
```